applications/RomApplication/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)

```

Comment:
@Rbravo555 This wile changed significantly (simplified), I recommend checking out the template app and updating it here, will make your life easier in the future

Reply:
@philbucher Following your recommendation I have updated the file

Reply:
:+1: